src/components/CategoryFilter.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { CHANGE_FILTER } from '../actions/index';
+
+const CategoryFilter = ({ CHANGE_FILTER }) => {
+  const categories = [
+    'All',
```

Comment:
- [ ] Since the CategoryFilter component is supposed to be a presentational component it must not only be placed at `src/components` directory but also needs to be unaware of Redux existence, therefore, does not connect with Redux. You can read more about the Redux approach to separate components into presentational and container components [here](https://redux.js.org/basics/usage-with-react#presentational-and-container-components). What we need to do instead is implement `handleFilterChange` at the `BookList` component (which is a container component, thus, it is connected with Redux) and pass `handleFilterChange` as a callback prop to the `CategoryFilter` component.
![Screenshot from 2020-10-06 08-57-11](https://user-images.githubusercontent.com/30977845/95198666-f838a600-07b1-11eb-8a03-3c1f65bc9833.png)